python/ray/rllib/evaluation/metrics.py

```diff
@@ -59,5 +59,4 @@ def summarize_episodes(episodes):
         episode_reward_min=min_reward,
         episode_reward_mean=avg_reward,
         episode_len_mean=avg_length,
-        episodes_total=len(episode_lengths),
```

Comment:
This was misleading since it was smoothed over the last 100 episodes, and was always 100 as a result...